ci-operator/config/openshift/cert-manager-operator/openshift-cert-manager-operator-cert-manager-1.18.yaml

```diff
@@ -0,0 +1,147 @@
+base_images:
+  base-rhel9:
+    name: "4.20"
+    namespace: ocp
+    tag: base-rhel9
+  cli-operator-sdk:
+    name: cli-operator-sdk
+    namespace: ocp
+    tag: v1.39.2
+  ocp_builder_rhel-9-golang-1.24-openshift-4.20:
+    name: builder
+    namespace: ocp
+    tag: rhel-9-golang-1.24-openshift-4.20
+binary_build_commands: make build --warn-undefined-variables
+build_root:
+  from_repository: true
+images:
+- dockerfile_path: images/ci/Dockerfile
+  from: base-rhel9
+  inputs:
+    ocp_builder_rhel-9-golang-1.24-openshift-4.20:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.24-openshift-4.20
+  to: cert-manager-operator
+- dockerfile_path: images/ci/operand.Dockerfile
+  from: base-rhel9
+  inputs:
+    ocp_builder_rhel-9-golang-1.24-openshift-4.20:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.24-openshift-4.20
+  to: cert-manager
+operator:
+  bundles:
+  - as: cert-manager-operator-bundle
+    dockerfile_path: bundle.Dockerfile
+    skip_building_index: true
+  substitutions:
+  - pullspec: openshift.io/cert-manager-operator:latest
+    with: pipeline:cert-manager-operator
+  - pullspec: quay.io/jetstack/cert-manager-controller:latest
+    with: pipeline:cert-manager
+  - pullspec: quay.io/jetstack/cert-manager-webhook:latest
+    with: pipeline:cert-manager
+  - pullspec: quay.io/jetstack/cert-manager-cainjector:latest
+    with: pipeline:cert-manager
+releases:
+  latest:
+    candidate:
+      architecture: multi
+      product: ocp
+      stream: nightly
+      version: "4.19"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: verify
+  commands: |
+    make verify -k --warn-undefined-variables
+  container:
+    from: src
+- as: verify-deps
+  commands: |
+    make verify-deps -k --warn-undefined-variables
+  container:
+    from: src
+- as: unit
+  commands: |
+    make test --warn-undefined-variables
+  container:
+    from: src
+- as: fips-image-scan-operator
+  steps:
+    dependencies:
+      SCAN_IMAGE: cert-manager-operator
+    test:
+    - ref: fips-check-image-scan
+- as: fips-image-scan-cert-manager
+  steps:
+    dependencies:
+      SCAN_IMAGE: cert-manager
+    test:
+    - ref: fips-check-image-scan
+- as: e2e-operator
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    owner: openshift-ci
+    product: ocp
+    timeout: 2h0m0s
+    version: "4.19"
```

Comment:
```suggestion
    version: "4.20"
```

Reply:
Should we modify in master branch as well? https://github.com/openshift/release/blob/10413e49e5613d744beedcf13637f8d4b4dc8ee6/ci-operator/config/openshift/cert-manager-operator/openshift-cert-manager-operator-master.yaml#L52